Quoteau/Podfile

```diff
@@ -0,0 +1,12 @@
+# Uncomment the next line to define a global platform for your project
```

Comment:
There is a warning while running pod install: 
[!] Automatically assigning platform `iOS` with version `13.1` on target `Quoteau` because no platform was specified. Please specify a platform for this target in your Podfile. See `https://guides.cocoapods.org/syntax/podfile.html#platform`.